data/map/stairs.yml

```
@@ -351,21 +479,233 @@
+  tile:
+    x: 3140
+    y: 3305
     plane: 1
-- id: 45482
+  delta:
+    plane: -1
+- id: 36795
+  option: Climb-up
+  tile:
+    x: 3164
+    y: 3307
+  delta:
+    plane: 1 
+- id: 36796
+  option: Climb-up
+  tile:
+    x: 3164
+    y: 3307
+    plane: 1
+  delta:
+    plane: 1     
+- id: 36796
   option: Climb-down
   tile:
-    x: 3215
-    y: 3239
+    x: 3164
+    y: 3307
     plane: 1
   delta:
-    x: 3
-    plane: -1
\ No newline at end of file
+    plane: -1
+- id: 36797
+  option: Climb-down
+  tile:
+    x: 3164
+    y: 3307
+    plane: 2
+  delta:
+    plane: -1    
+#wizard tower
+- id: 2147
+  option: Climb-down
+  tile:
+    x: 3104
+    y: 3162
+  delta:
+    y: 6415
+- id: 32015
+  option: Climb-up
+  tile:
+    x: 3103
+    y: 9576
+  delta:
+    y: -6415
+- id: 12536
+  option: Climb-up
+  tile:
+    x: 3103
+    y: 3159
+  delta:
+    x: -1
+    y: 1
+    plane: 1
+- id: 12537
+  option: Climb-down
+  tile:
+    x: 3103
+    y: 3159
+    plane: 1
+  delta:
+    x: 1
+    y: -1
+    plane: -1    
+- id: 12537
+  option: Climb-up
+  tile:
+    x: 3103
+    y: 3159
+    plane: 1
+  delta:
+    plane: 1
+- id: 12538
+  option: Climb-down
+  tile:
+    x: 3104
+    y: 3160
+    plane: 2
+  delta:
+    plane: -1    
+# draynor
+- id: 7057
+  option: Climb-up
+  tile:
+    x: 3090
+    y: 3251
+  delta:
+    x: 4
+    plane: 1
+- id: 7056
+  option: Climb-down
+  tile:
+    x: 3091
+    y: 3251
+    plane: 1
+  delta:
+    x: -4
+    plane: -1
+- id: 46244
+  option: Climb-up
+  tile:
+    x: 3099
+    y: 3266
+  delta:
+    x: 4
+    plane: 1
+- id: 46245
+  option: Climb-down
+  tile:
+    x: 3100
+    y: 3266
+    plane: 1
+  delta:
+    x: -4
+    plane: -1   
+# draynor Manor   
+- id: 47364
+  option: Climb-up
+  tile:
+    x: 3107
+    y: 3362
+  delta:
+    y: 5
+    plane: 1
+- id: 47657
+  option: Climb-down
+  tile:
+    x: 3108
+    y: 3364
+    plane: 1
+  delta:
+    y: -5
+    plane: -1 
+- id: 47574
+  option: Climb-up
+  tile:
+    x: 3105
+    y: 3363
+    plane: 1
+  delta:
+    y: -2
+    plane: 1
+- id: 47575
+  option: Climb-down
+  tile:
+    x: 3105
+    y: 3363
+    plane: 2
+  delta:
+    y: 2
+    plane: -1 
+- id: 47643
+  option: Walk-down
+  tile:
+    x: 3115
+    y: 3356
+  delta:
+    x: -35
+    y: 6421
+- id: 164
+  option: Climb-up
+  tile:
+    x: 3080
+    y: 9772
+  delta:
+    x: 35
+    y: -6421 
+- id: 133
+  option: Climb-down
+  tile:
+    x: 3092
+    y: 3362
+  delta:
+    x: 25 
+    y: 6392    
+- id: 32015
+  option: Climb-up
+  tile:
+    x: 3117
+    y: 9754
+  delta:
+    x: -25
+    y: -6392   
+# draynor trapdoors
+- id: 6435
+  option: Climb-down
+  tile:
+    x: 3084
+    y: 3272
+  delta:   
+    y: 6400 
+- id: 32015
+  option: Climb-up
+  tile:
+    x: 3084
+    y: 9672
+  delta:
+    y: -6400 
+- id: 6435
+  option: Climb-down
+  tile:
+    x: 3118
+    y: 3244
+  to:
+    x: 3118
+    y: 9644
+- id: 26518
+  option: Climb-up
+  tile:
+    x: 3118
+    y: 9643
```

```suggestion
    y: 9644
```
Looks like it should be 9644 instead of 9643?

---

if its 9644 i go there and i can't use it is it different for you?

---

Yeah you're correct, it's the trapdoor to: 9644 which needs to be 9643